django/db/backends/utils.py

```diff
@@ -209,3 +209,16 @@ def format_number(value, max_digits, decimal_places):
     if decimal_places is not None:
         return "%.*f" % (decimal_places, value)
     return "{:f}".format(value)
+
+
+def strip_trailing_quotes(table):
+    """
+    Prepares table name for tables, triggers, sequences.
+    It processes case when db_table option for model Meta contains double-quoted table name.
+    Resulting name will be returned without double quotes despite of it had it or not before preparing.
+    """
+
+    if table.startswith('"') and table.endswith('"'):
+        return table[1:-1]
```

Comment:
What about situation when `table` contains `username` e.g. `'"GEO"."DJANGO_PESSOA"'` (like in ticket)?